docs/cli.rst

```diff
@@ -53,6 +122,9 @@ Stream data from files in the `Karabo bridge
 <https://rtd.xfel.eu/docs/data-analysis-user-documentation/en/latest/online.html#streaming-from-karabo-bridge>`_
 format. See :doc:`streaming` for more information.
 
+For streaming data from a run directory, we recommend the newer
+:ref:`cmd-serve-run` command in place of this.
+
```

Comment:
In fact, I have used the old command (only) for streaming from a run directory, using the full path as argument. Apart from the fact that ``...-serve-run`` is indeed more convenient to achieve this, what would be the main use case for using the old command now?

Reply:
The `-serve-files` command makes it easy to stream from a non-standard run directory location, e.g. if we do an experimental correction of a run, we might put it in proposal `scratch`. Or it gives you a way to stream from `red` before we've integrated support for that. Or if users transfer run data back to their home institution and want to use EXtra-data there. I think the new `-serve-run` command will be better for ~95% of use cases.

The biggest reason to retain the old command is compatibility & familiarity, though - don't break what's working for people. :slightly_smiling_face: